ecs-logging-core/src/main/java/co/elastic/logging/TimestampSerializer.java

```diff
@@ -110,9 +112,10 @@ private void serializeWithLeadingZero(StringBuilder builder, long value, int min
         private final long endOfCachedDate;
 
         private CachedDate(long epochTimestamp) {
-            SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd", Locale.ROOT);
-            dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
-            cachedDateIso = dateFormat.format(new Date(epochTimestamp));
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+                    .localizedBy(Locale.ROOT)
+                    .withZone(ZoneId.of("UTC"));
+            cachedDateIso = formatter.format(Instant.ofEpochMilli(epochTimestamp));
```

Comment:
Uses APIs that are not available in Java 6

Reply:
Hi, sorry about that. I've reverted the incompatible changes.

/Peldan